src/Traits/LogsActivity.php

```diff
@@ -3,12 +3,9 @@
 namespace Backpack\ActivityLog\Traits;
 
 use Spatie\Activitylog\LogOptions;
-use Spatie\Activitylog\Traits\LogsActivity as OriginalLogsActivity;
 
 trait LogsActivity
 {
-    use OriginalLogsActivity;
```

Comment:
@promatik I don't understand this.

Before this PR - my models were using `Backpack\ActivityLog\Traits\LogsActivity`, so they were logging everything the users did, with the Backpack defaults.

After this PR - my models are still using `Backpack\ActivityLog\Traits\LogsActivity` but they aren't logging anything any more.

---

To make it work again, now my models have to use both Spatie's `LogActivity` trait and Backpack's `LogsActivity` trait? Is this intentional? Why?

Reply:
Omg 😅
Please ignore this change, I was removing an override in the trait, and end up remove wrongly, the whole trait.
Reverting...